src/module_3/predict.py

```diff
@@ -0,0 +1,21 @@
+from joblib import load
+from train import OUTPUT_PATH, evaluate_model, clean_dataset
+import pandas as pd
+
+def main():
+    model_name = '/20241104193743_logistic_regression.pkl'
```

Comment:
Maybe we could make an input instead of hardcoding the model binaries. This doesn't take much advantage of reproducibility we get from a modular implementation